ibm/resource_ibm_iam_access_group_members.go

```diff
@@ -262,7 +269,8 @@ func resourceIBMIAMAccessGroupMembersUpdate(context context.Context, d *schema.R
 			if err != nil {
 				return diag.FromErr(fmt.Errorf("ERROR] Error Getting Service Ids %s %s", err, resp))
 			}
-			err = iamuumClient.AccessGroupMember().Remove(grpID, *serviceID.IamID)
+			removeMembersFromAccessGroupOptions := iamAccessGroupsClient.NewRemoveMemberFromAccessGroupOptions(grpID, *serviceID.IamID)
+			_, err = iamAccessGroupsClient.RemoveMemberFromAccessGroup(removeMembersFromAccessGroupOptions)
 			if err != nil {
 				return diag.FromErr(err)
```

Comment:
return diag.FromErr(err) ===> return diag.FromErr(fmt.Errorf("Error removing members to group(%s). API response: %s", grpID, detailResponse))